README.md

```diff
@@ -1,3 +1,431 @@
-# example-files
+# XDF example files
+
+Example files to test XDF readers.
+
+* [minimal.xdf](#minimal)
+* [clock\_resets.xdf](#clock_resets)
+* [empty\_streams.xdf](#empty_streams)
+
+## minimal.xdf <a name="minimal"></a>
```

Comment:
Setting a different anchor doesn't seem to work (at least when looking at the version rendered on GitHub). When I click at the link to the left of the heading, the anchor is `#minimalxdf-`. All of this is to say that I wouldn't bother changing anchors, just leave them at their defaults.